<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitoringLeftPanel</class>
 <widget class="QWidget" name="MonitoringLeftPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>242</width>
    <height>38</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLabel" name="resultControlComplexLabel">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Результат контроля работоспособности</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid grey;
background-color: grey;
QToolButton { color: red};</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultControlDeviceLabel">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Результат контроля функционирования аппаратных средств</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid grey;
background-color: grey;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultControlSoftLabel">
     <property name="minimumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
     <property name="toolTip">
      <string>Результат контроля программного обеспечения комплекса</string>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 10px;
border: 1px solid grey;
background-color: grey;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
